src/views/Watch.vue

```diff
@@ -212,20 +209,11 @@ export default {
             playlistIndex: -1,
             currentTime: 0,
             player: null,
-            theaterMode: false,
-            altTHotKey: [
-                {
-                    keyCode: 84, // T
-                    modifiers: ["alt"],
-                    preventDefault: false,
-                },
-            ],
-
         };
     },
     computed: {
         ...mapState("watch", ["video", "isLoading", "hasError"]),
-        ...syncState("watch", ["showTL", "showLiveChat"]),
+        ...syncState("watch", ["showTL", "showLiveChat", "theaterMode"]),
```

Comment:
huh does this actually work?

the syncState generates setters i thought :o

Reply:
i actually forgot to test the behavior without this addition since it's the first thing i did... but from what I recall it didn't break anything?

Reply:
it *shouldn't* be just this much changes, shouldn't modifying syncState'd variables go through `setTheaterMode` or something...? 🤔

Reply:
i think the real fix was actually removing `theaterMode: false` from default, there's a decent chance that syncstate change did nothing. I'll try test

Reply:
syncState actually syncs against the `theaterMode` key inside `watch` store, but take a look how the other toggles are memorized. There's a good chance this doesn't work as your PR says on the tin.

Reply:
Ok so removing it from syncstate broke it, but from both my original testing and what I tested just now the original code really does persist through page loads without any other modifications. Perhaps it should be added to anyway?

Reply:
okay LOL